recipes/blpapi/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "blpapi" %}
+{% set version = "3.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://bloomberg.bintray.com/pip/simple/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 4c3cbf3fb63fd338360d55188f6a59568023be6e16531d4bd307ac59a297fd96
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-linux.tar.gz
+    sha256: c6c425f714b461ec618e6900bda8d9b4fc919892538ccfbf8c9a774e9e234fee
+    folder: blpapi_cpp_linux
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-windows.zip
+    sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec
+    folder: blpapi_cpp_windows
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python <=3.6
```

Comment:
Please remove this ``<=3.6``, in staged recipes we are using just python 2.7, 3.4, 3.5 and 3.6

Reply:
Just to confirm, I should not include this even if I know `python=3.7` will break this?